packages/packagers/js/src/prelude.js

```diff
@@ -10,7 +10,10 @@
 parcelRequire = function(modules, cache, entry, globalName) {
   // Save the require from previous bundle to this closure if any
   var previousRequire = typeof parcelRequire === 'function' && parcelRequire;
-  var nodeRequire = typeof require === 'function' && require;
+  var nodeRequire =
+    typeof module !== 'undefined' &&
+    typeof module.require === 'function' &&
+    module.require.bind(module);
```

Comment:
Can you put a comment about why we don't just use `require` so someone doesn't try to change this later?